### DISCUSSION TITLE
Visual Studio Code

### DISCUSSION THREAD

**Discussion by AracelyT** [community/community#56990] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Cuál es el mejor lenguaje de programación para trabajar en Visual Studio Code?

**Comment by BLKKKBVSIK** [comment]
Hi @AracelyT,
Welcome on Github Community Discussions,

To answer your question, Visual Studio Code is a text editor that work with a lot of programming language.
It's impossible to say which programming language is the best to work with Visual Studio Code.
If the language you want to work with is not supported by default, you might find an extension for Visual Studio Code that will bring the support for this specific language.

You can find all those extensions with the link bellow:
https://marketplace.visualstudio.com/search?target=VSCode&category=Programming%20Languages&sortBy=Installs
